kaska/watercloudmodel.py

```diff
@@ -20,10 +20,11 @@
 (`A`, `B`, `C`, `D`) are polarisation dependent. `V1` and `V2` have to do with
 the scatterers within the turbid medium, and are usually related to LAI.
 """
+# pylint: enable=anomalous-backslash-in-string
 
 import numpy as np
 
-from numba import jit
+# from numba import jit  # unused import
```

Comment:
Delete

Reply:
Zen of Python https://www.python.org/dev/peps/pep-0020/ number 2 - Explicit is better than implicit.
So it seems better to leave it in.